# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: Cargo.toml
# Commit: 1d5a44ae7207
# Full Hash: 1d5a44ae72071f079dcb00b62419c2ddd3c9f12c
# Author: Norisz Fay <nfay@mozilla.com>
# Date: 2023-01-27 21:11 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing build bustages on DecoderData.cpp CLOSED TREE
# ==============================================================================

diff -r 83609f9575ac -r 1d5a44ae7207 Cargo.toml
--- a/Cargo.toml	Fri Jan 27 19:06:16 2023 +0000
+++ b/Cargo.toml	Fri Jan 27 21:11:42 2023 +0200
@@ -104,6 +104,9 @@
 # Patch getrandom 0.7 to 0.8
 rand = { path = "build/rust/rand" }
 
+# Patch env_logger 0.8 to 0.9
+env_logger = { path = "build/rust/env_logger" }
+
 # Patch parking_lot 0.12 down to 0.11, which is compatible for most crates that use it, to avoid
 # dependencies on windows-sys.
 parking_lot = { path = "build/rust/parking_lot" }